packages/fuel-indexer-tests/tests/e2e/indexing_postgres.rs

```diff
@@ -484,3 +484,58 @@ async fn test_index_metadata_is_saved_when_indexer_macro_is_called_postgres() {
     assert!(block_height >= 1);
     assert!(time >= 1);
 }
+
+#[actix_web::test]
+#[cfg(all(feature = "e2e", feature = "postgres"))]
+async fn test_index_not_triggered_when_start_block_not_met_postgres() {
```

Comment:
@deekerno  
- Nice stuff
- So this tests the case where items should not be saved due to `CURRENT_NODE_BLOCK_HEIGHT < MANIFEST_BLOCK_HEIGHT` 👍 
- But it doesn't test the case where `CURRENT_NODE_BLOCK_HEIGHT >= MANIFEST_BLOCK_HEIGHT`
- Should be pretty simple to add:
  - If you set your `start_block` to something small (e.g., `4`) you could test the case where items are not indexed _and_ the case where items are indexed (after that start block has been reached)